assets/scss/_grpc.scss

```diff
@@ -229,17 +234,24 @@ c - Component (Aware of its content/context...)
 // Object
 .o-banner {
   width: 100%;
-  position: fixed;
-  margin-left: -15px;
+  position: relative;
   padding-top: 15px;
-  z-index: 32;
+  // keep banner's z-index value less than the navbar, so that banner moves beneath the navbar when user scrolls the page
+  z-index: $navbar-z-index - 1;
   top: 4rem;
   background: $light;
   text-align: center;
 
   & p {
     padding: 0.5rem;
   }
+  padding-bottom: 0.01rem;
+}
+
+@include media-breakpoint-down(sm){
+  .o-banner{
+    top: 0rem;
```

Comment:
Can you add a comment explaining that this is required because the navbar changes `position` from `fixed` to `absolute`. This behaviour will not be obvious to future readers.